xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -947,6 +982,16 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Indicates a tactile marker on floor under the buttons (or aligned with).</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="MonitoringRemoteControl" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether the lift is equipped with a remote operation control system</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="OtherDoors" type="OtherDoorsEnumeration" default="same" minOccurs="0">
```

Comment:
```suggestion

```

Reply:
Door positions can be described using *DoorOrientation* in *LiftCallEquipment*, see #634.